test/Python/python_runtime.swift

```diff
@@ -285,6 +288,10 @@ PythonRuntimeTestSuite.testWithLeakChecking("ConvertibleFromPython") {
 
   expectEqual("abc", String(string))
 
+  expectEqual([2, 3], Array(intArray))
+  expectEqual(TensorShape(2, 3), TensorShape(intArray))
+  expectEqual(["abc": 97], Dictionary<String, Int32>(dict))
```

Comment:
In the Swift core repo we put a space before `:`. This is a bit different from tensorflow/swift-apis.

```suggestion
  expectEqual(["abc" : 97], Dictionary<String, Int32>(dict))
```